docker-compose.dev.yml

```diff
@@ -1,15 +1,16 @@
 version: '3.8'
 
+name: transcendence-dev
```

Comment:
Cette ligne fait planter le `make dev` chez moi, avec cette erreur : 

` rmorel  ~/ft_transcendence   setup-api-orm  make dev
docker compose --env-file .env -f docker-compose.dev.yml build
(root) Additional property name is not allowed
make: *** [Makefile:32: dev.build] Error 15`